src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputOperatorWrap.java

```diff
@@ -104,39 +104,37 @@ class Inner {
             for (Map.Entry<String, String> entry :
                 map.entrySet())
             {
-                /*warn*/int i = flag == true ?
+                int i = flag == true ? // violation ''?' should be on a new line.'
                         1 : 2;
             }
 
-            /*warn*/ if (init !=
+             if (init != // violation ''!=' should be on a new line.'
                     9)
             {
 
             }
 
-            /*warn*/ while (init ==
+             while (init == // violation ''==' should be on a new line.'
                     10)
             {
 
             }
 
-            /*warn*/ if (init >
+             if (init > // violation ''>' should be on a new line.'
                     10)
             {
 
             } else {}
 
-            /*warn*/ while (init < 10 ||
+             while (init < 10 || // violation '.* should be on a new line.'
                     !flag) {
 
             }
         }
     }
 
-    Inner anon = new Inner
-            (){
-        void testGenerics1
-        ()
+    Inner anon = new Inner(){
+        void testGenerics1()
```

Comment:
Reformatted the code cuz it was related to `MethodParamPad` module, the original code has been tested in its own separate file. This file contains code only related to `OperatorWrap` module